src/sshagent/SSHAgent.cpp

```diff
@@ -33,6 +33,8 @@ SSHAgent::SSHAgent(QObject* parent)
 {
 #ifndef Q_OS_WIN
     m_socketPath = QProcessEnvironment::systemEnvironment().value("SSH_AUTH_SOCK");
+#else
+    m_socketPath = "\\\\.\\pipe\\openssh-ssh-agent";
```

Comment:
Do you want to consider only falling back to "\\\\.\\pipe\\openssh-ssh-agent" when SSH_AUTH_SOCK is not defined? Since for OpenSSH-portable for windows, SSH_AUTH_SOCK will still be checked.